packages/expo-calendar/android/src/main/java/expo/modules/calendar/next/utils/CalendarNextUtils.kt

```diff
@@ -11,12 +11,36 @@ import expo.modules.calendar.next.exceptions.EventDateTimeInvalidException
 import java.text.SimpleDateFormat
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
+import java.util.Date
 
 val sdf = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").apply {
   timeZone = TimeZone.getTimeZone("GMT")
 }
+
+/**
+ * [SimpleDateFormat] used in native recurrence rule string.
+ * The format corresponds to the 'date-time' type defined by RFC-5455 section 3.3.5.
+ */
 val rrFormat = SimpleDateFormat("yyyyMMdd'T'HHmmss'Z'")
```

Comment:
Shouldn't the TimeZone also be applied here?
```suggestion
val rrFormat = SimpleDateFormat("yyyyMMdd'T'HHmmss'Z'").apply {
  timeZone = TimeZone.getTimeZone("GMT")
}
```